iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -106,6 +113,26 @@ public override async Task OpenAsync(CancellationToken cancellationToken)
             }
         }
 
+        private void checkTimeout(Object _)
+        {
+            if (_twinResponseTimeouts.Any())
+            {
+                var currentDateTime = DateTime.UtcNow;
+                TimeSpan difference;
```

Comment:
Why declare this outside the foreach?

Reply:
Just to avoid create a new variable for each key value pair. This way we can re-use it.

Reply:
Not really saving anything, I think.

It makes it look like we're saving the value between iterations, so for code readability reasons, I'd inline it.